card/src/main/java/com/adyen/checkout/card/internal/ui/view/CardIcons.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2025 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ozgur on 2/12/2025.
+ */
+
+package com.adyen.checkout.card.internal.ui.view
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import com.adyen.checkout.card.R
+import com.adyen.checkout.ui.internal.isDarkMode
+
+@Composable
+internal fun rememberCardIcons(): CardIcons {
+    val isDark = isDarkMode()
+    return remember(isDark) {
+        if (isDark) CardIcons.Dark else CardIcons.Light
+    }
+}
```

Comment:
Together with the other comment I think we could change this function into something like:
```suggestion
@Stable
@Composable
internal fun getThemedIcon(
    backgroundColor: Color,
    lightDrawableId: Int,
    darkDrawableId: Int,
): Int {
    return remember(backgroundColor) {
        if (backgroundColor.isDark()) darkDrawableId else lightDrawableId
    }
}
```
This would make the function more generic and reusable in other places. It would also make sure you don't get more res ids than you really need